css/style.css

```diff
@@ -1 +1,1461 @@
+  font-style: normal;
+  font-size: 18px;
+  line-height: 28px;
+  color: rgba(255, 255, 255, 1);
+}
+
+a {
+  text-decoration: none;
+  color: rgba(255, 255, 255, 1);
+}
+
+ul {
+  list-style-type: none;
+  padding: 0;
+  margin: 0;
+}
+
+.container {
+  padding: 0 255px;
+}
+
+/**************** Buttons styles ****************/
+
+button {
+  cursor: pointer;
+}
+
+.basic-btn {
+  padding: 12px 24px;
+  font-size: 16px;
+  line-height: 24px;
+  gap: 8px;
+}
+
+.media-btn img {
+  width: 24px;
+  height: 24px;
+  background-color: transparent;
+}
+
+.btn-light {
+  border: none;
+  background-color: var(--btn-primary-color);
+  color: var(--secondary-color);
+  font-family: var(--btn-header-font);
+}
+
+.btn-border {
+  border: 2px solid var(--btn-primary-color);
+  background-color: inherit;
+  color: rgba(255, 255, 255, 1);
+  font-family: var(--btn-header-font);
+}
+
+.btn-border:active {
+  background-color: rgba(33, 231, 134, 0.2);
+  transition: all 0.3s ease-in-out;
+}
+
+/* Text styles */
+.highlighted-text {
+  color: var(--btn-primary-color);
+}
+
+.secondary-heading {
+  font-size: 44px;
+  line-height: 56px;
+  text-align: center;
+}
+
+/********************* Header styles *****************************/
+
+header {
+  margin: 30px 0;
+}
+
+.logo-block {
+  font-size: 32px;
+  line-height: 42px;
+  gap: 16px;
+}
+
+.logo-block img {
+  width: 36px;
+  height: 40px;
+}
+
+/************* Header navigation styles *******************/
+
+.nav-menu-items {
+  gap: 32px;
+}
+
+.nav-menu-item a {
+  font-size: 16px;
+  line-height: 24px;
+  letter-spacing: 10%;
+}
+
+.arrow-dwn {
+  width: 15px;
+  height: 15px;
+  margin-left: 5px;
+}
+
+.dropdown-content a {
+  font-size: 16px;
+  line-height: 24px;
+  letter-spacing: 10%;
+}
+
+.btn-box {
+  gap: 20px;
+}
+
+.switch-mode {
+  width: 48px;
+  height: 48px;
+  margin: 0 auto;
+  background-color: transparent;
+  border: none;
+}
+
+.switch-mode .switch-ico {
+  width: 24px;
+  height: 24px;
+  padding: 12px;
+  border-radius: 41px;
+  background-color: var(--secondary-color);
+  fill: var(--btn-primary-color);
+  transition: all 0.3s ease-in-out;
+}
+
+.switch-mode .switch-ico:active {
+  background-color: var(--btn-primary-color);
+  fill: var(--secondary-color);
+}
+
+header .media-btn {
+  background-color: var(--secondary-color);
+  transition: color 0.3s ease-in-out;
+}
+
+header .media-btn:active {
+  background-color: rgb(35, 46, 57);
+  border: 2px solid var(--secondary-color);
+}
+
+.discord-btn {
+  width: 24px;
+  height: 24px;
+}
+
+.wallet-icon {
+  width: 24px;
+  height: 24px;
+}
+
+/**************** Hero section **************************/
+
+.hero-box {
+  padding-top: 141px;
+  background-image: url(../img/bg-s/star-big.svg),
+    linear-gradient(to top, rgba(4, 11, 17, 1), rgba(4, 11, 17, 0)),
+    radial-gradient(
+      ellipse at center,
+      rgba(48, 113, 101, 0.3) 10%,
+      rgba(20, 27, 34, 0.3) 70%
+    );
+  background-size: 496.05px auto, cover, 950px 278px;
```

Comment:
do not use fractional numbers, the browser rounds them

Reply:
#fixed